packages/kit/test/apps/basics/src/routes/load/invalidation/params/+layout.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+	/** @type {import('./$types').LayoutData} */
+	export let data;
+</script>
+
+<slot />
+
+<pre>{JSON.stringify({
+		a: data.a,
+		b: data.b
+	})}</pre>
```

Comment:
```suggestion
<pre>{JSON.stringify(data)}</pre>
```

Reply:
it includes a bunch of stuff from the root layout if we do that

Reply:
Oh right, I forgot about the merge thing 👍